apps/remix-ide/src/app/files/remixd-handle.js

```diff
@@ -145,6 +145,9 @@ function remixdDialog () {
         <i class="fas fa-link"></i> will show you current connection status.
       </div>
       <div class=${css.dialogParagraph}>This feature is still in Alpha, so we recommend you to keep a copy of the shared folder.</div>
+      <span class="text-danger">
+        Please make sure you have the <b>latest remixd version</b>. <a target="_blank" href="https://remix-ide.readthedocs.io/en/latest/remixd.html?highlight=remixd#update-to-the-latest-remixd">Read here how to update it</a>
```

Comment:
The hightlight param in the link makes the page very noisy - try removing it:
https://remix-ide.readthedocs.io/en/latest/remixd.html#update-to-the-latest-remixd